packages/scalecube-microservice/src/helpers/validation.ts

```diff
@@ -0,0 +1,95 @@
+import * as check from './check';
+import {
+  SEED_ADDRESS_IS_NOT_STRING,
+  SERVICES_IS_NOT_ARRAY,
+  SERVICE_IS_NOT_OBJECT,
+  MICROSERVICE_OPTIONS_IS_NOT_OBJECT,
+  SERVICE_DEFINITION_NOT_PROVIDED,
+  SERVICE_NAME_NOT_PROVIDED,
+  DEFINITION_MISSING_METHODS,
+  INVALID_METHODS,
+  MESSAGE_NOT_PROVIDED,
+  WRONG_DATA_FORMAT_IN_MESSAGE,
+  QUALIFIER_IS_NOT_STRING,
+  INVALID_QUALIFIER,
+  INVALID_MESSAGE,
+  MESSAGE_QUALIFIER_NOT_PROVIDED,
+  MESSAGE_DATA_NOT_PROVIDED,
+  getServiceNameInvalid,
+  ASYNC_MODEL_TYPES,
+  getIncorrectMethodValueError,
+  getInvalidMethodReferenceError,
+  getAsynModelNotProvidedError,
+  getInvalidAsyncModelError,
+  getInvalidServiceReferenceError,
+  getServiceReferenceNotProvidedError,
+} from './constants';
+import { getQualifier } from '../helpers/serviceData';
+import ServiceDefinition from '../api/ServiceDefinition';
+
+export const validateMicroserviceOptions = (microserviceOptions: any) => {
```

Comment:
```suggestion
export const validateMicroserviceOptions = (microserviceOptions: MicroserviceOptions) => {
```

Reply:
microserviceOptions here has 'any' type by design since validation function shoud have possiblity to check input value of any type